libs/s25main/ingameWindows/IngameWindow.cpp

```diff
@@ -230,16 +233,24 @@ void IngameWindow::MouseLeftUp(const MouseCoords& mc)
 
 void IngameWindow::MouseMove(const MouseCoords& mc)
 {
-    // Fenster bewegen, wenn die Bewegung aktiviert wurde
     if(isMoving)
     {
-        DrawPoint newPos = GetPos() + mc.GetPos() - lastMousePos;
+        // Calculate window rectangle without snapping
+        DrawPoint delta = mc.GetPos() - lastMousePos;
+        DrawPoint oldPos = GetPos() - snapOffset_;
+        Rect wndRect(oldPos, GetSize());
```

Comment:
Similar question: is this the draw rect or the boundary rect? When are they different?

Reply:
I'd use the boundary rect.